var/spack/repos/builtin/packages/coreneuron/package.py

```diff
@@ -19,7 +19,8 @@ class Coreneuron(CMakePackage):
     git      = "https://github.com/BlueBrain/CoreNeuron"
 
     version('develop', branch='master', submodules=True)
-    version('0.22', tag='0.22', submodules=True, preferred=True)
+    version('0.23a', commit="c5f6563", submodules=True, preferred=True)
```

Comment:
preferred=True not required and makes the line to diff every time there's a bump. Ok now, let's avoid a rebuild, but next time

Reply:
Agreed, let's leave it this time, otherwise we will need to wait another 10 hours for Travis CI to start the job ;).